bridge/slack/handlers.go

```diff
@@ -290,6 +298,27 @@ func (b *Bslack) handleMessageEvent(ev *slack.MessageEvent) (*config.Message, er
 	return &rmsg, nil
 }
 
+func (b *Bslack) handleTypingEvent(ev *slack.UserTypingEvent) (*config.Message, error) {
+	var err error
+	// use our own func because rtm.GetChannelInfo doesn't work for private channels
+	channelInfo, err := b.getChannelByID(ev.Channel)
+	if err != nil {
+		return nil, err
+	}
+
+	rmsg := config.Message{
+		Username: b.getUsername(ev.User),
```

Comment:
Username not needed, not getting used anywhere ?

Reply:
I think you should also exclude the USER_TYPING event there from getting logged as it's too noisy (like you did on the other places), so it's not needed I think